app/components/user-status.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="heading">
+  {{#if (eq @status "active")}}
+    <h2 data-test-status>I am doing a task</h2>
+  {{/if}}
+  {{#if (eq @status "idle")}}
+    <h2 data-test-status>I am idle</h2>
+  {{/if}}
+  {{#if (eq @status "ooo")}}
+    <h2 data-test-status>I am OOO</h2>
+  {{/if}}

```

Comment:
At least line 2-10 can be easily simplified by using an array of objects, you'll have to add the object in a component.js file
```JS
[
  {
    status: "active"
    text: "I'm doing a task
  },
  // Hope you get the idea, check how tasks page is reusing an array of objects recieved from API
```

Reply:
While looping over the array, you'll check if the current status value (from API) is equal to object you're currently iterating on

Reply:
ok got it, will implement this.